### PR TITLE
added API endpoints to api_setup.py and modified app.py

### PR DESCRIPTION
**Pull request by alanira**

### CODE REVIEW

File: server/api_setup.py [hunk 1]
```diff
@@ -1,17 +1,35 @@
 """This module is an API set up
 SHOULD WORK WITH YELP API"""
+
 import os
 import random
 import requests
 import sys
 from dotenv import find_dotenv, load_dotenv
 
+print(find_dotenv())
```
Comment (nh124) [inline server/api_setup.py:10]: Maybe remove this print statement

**Review by nh124 — COMMENTED** [review]
I think we lint our code each time we make a pull request.

File: server/api_setup.py (hunk 1 above)
Comment (alanira) [reply server/api_setup.py:10]: I removed it.

**Review by nh124 — APPROVED** [review]
Great!

File: server/api_setup.py (hunk 1 above)
Comment (nh124) [reply server/api_setup.py:10]: Great! go ahead and merge. I have approved it.

**Pull request merged**